reboot_toolkit/reboot_motion_api.py

```diff
@@ -0,0 +1,289 @@
+from __future__ import annotations
+
+import gzip
+import json
+import os
+
+from concurrent.futures import ThreadPoolExecutor
+from io import BytesIO
+from itertools import repeat
+
+import pyarrow as pa
+import requests
+import warnings
+
+
+def locals_to_input(local_vars: dict) -> dict:
```

Comment:
do you have the link for where you saw that this would be the case? 

i've been using this locals() function just fine and i haven't seen any keys like this, but i'm curious to see the resource that mentioned this as a possibility

Reply:
@jimmybuffi I'm going to resolve this since we discussed on Slack, no need to modify anything here.